helpers/lib/fonts.js

```diff
@@ -18,35 +16,35 @@ const fontProviders = {
          * @returns {string}
          */
         parser: function(fonts) {
-            var collection = [],
-                familyHash = {};
+            const collection = [];
+            const familyHash = {};
 
-            _.each(fonts, function fontsIterator(font) {
-                var split = font.split('_'),
-                    familyKey = split[1],  // Eg: Open+Sans
-                    weights = split[2];    // Eg: 400,700italic
+            for (let i = 0; i < fonts.length; i++) {
+                const split = fonts[i].split('_');
+                const familyKey = split[1];
+                let weights = split[2];
 
-                if (_.isEmpty(familyKey)) {
-                    return;
+                if (!familyKey || !familyKey.length) {
+                   continue;
                 }
-
-                if (_.isUndefined(weights)) {
+                if (typeof weights === 'undefined') {
                     weights = '';
                 }
-
-                if (!_.isArray(familyHash[familyKey])) {
+                if (!Array.isArray(familyHash[familyKey])) {
                     familyHash[familyKey] = [];
                 }
-
                 weights = weights.split(',');
+                familyHash[familyKey].push(...weights);
+            }
 
-                familyHash[familyKey].push(weights);
-                familyHash[familyKey] = _.uniq(_.flatten(familyHash[familyKey]));
-            });
+            const familyHashKeys = Object.keys(familyHash);
 
-            _.each(familyHash, function fontHashIterator(weights, family) {
-                collection.push(family + ':' + weights.join(','));
-            });
+            for (let i = 0; i < familyHashKeys.length; i++) {
+                const family = familyHashKeys[i];
+                const weights = familyHash[family];
+                const uniqueWeights = Array.from(new Set(weights));
```

Comment:
@bc-williamkwon you can just set in weights the unique values instead of defining a new var